hs_core/templates/pages/genericresource.html

```diff
@@ -2674,12 +2675,14 @@ <h3 style="color: red" id="file-type-error"></h3>
                                 this.remove();
                             });
                             $(".alert-success").fadeTo(2000, 500).fadeOut(1000, function(){
+                                $(document).trigger(jQuery.Event("submit-error"));
```

Comment:
This was a mistake and has been corrected in 86e80edaf5996c5e688233fb64a69e690837cb16.  The new code raises a `submit-success` event.